polaris/polaris/transaction/views.py

```diff
@@ -74,13 +75,14 @@ def _construct_more_info_url(request):
 @xframe_options_exempt
 @api_view()
 @renderer_classes([TemplateHTMLRenderer])
-def more_info(request):
+@validate_sep10_token(content_type="text/html")
```

Comment:
How do we expect a JWT header to be sent to the more_info url?  It's just a url that's opened in a popup, often not something we can send headers to.

Reply:
Good catch, I'll remove it. However this endpoint should be protected somehow, since its showing transaction details. Maybe we use that one-time-token strategy here as well.